shared/chat/conversation/info-panel/container.js

```diff
@@ -116,73 +111,51 @@ const mapDispatchToProps = (dispatch: Dispatch) => ({
       ])
     )
   },
-  _onLeaveTeam: (teamname: TeamTypes.Teamname) =>
-    dispatch(navigateAppend([{props: {teamname}, selected: 'reallyLeaveTeam'}])),
   _onViewTeam: (teamname: TeamTypes.Teamname) =>
     dispatch(navigateTo([teamsTab, {props: {teamname: teamname}, selected: 'team'}])),
   onShowProfile: (username: string) => dispatch(createShowUserProfile({username})),
 })
 
-const mergeProps = (stateProps, dispatchProps, ownProps) => ({
+const mergeProps = (stateProps, dispatchProps, ownProps): InfoPanelProps => ({
```

Comment:
had to explicitly type this to get flow to check it properly...I suspect flow is losing the type info across files.